docusaurus.config.js

```diff
@@ -34,28 +32,17 @@ module.exports = {
       },
     ],
     "./static/plugins/bpmn-js",
-    [
-      "@docusaurus/plugin-content-docs",
-      {
-        id: "optimize",
-        path: "optimize",
-        routeBasePath: "optimize",
-        beforeDefaultRemarkPlugins: [versionedLinks],
-        sidebarPath: require.resolve("./optimize_sidebars.js"),
-        editUrl: "https://github.com/camunda/camunda-platform-docs/edit/main/",
-        versions: {
-          "3.9.0": {
-            banner: "none",
-          },
-          "3.8.0": {
-            banner: "none",
-          },
-        },
-      },
-    ],
   ],
   scripts: [],
   themeConfig: {
+    announcementBar: {
+      id: "camunda8",
```

Comment:
Should this be "unmaintained" to match line 216?

Reply:
I think they're unrelated! The `id` here is an identifier that I don't think is super significant; and line 216's `banner` property [can only take on one of three values[(https://docusaurus.io/docs/2.3.1/api/plugins/@docusaurus/plugin-content-docs#VersionsConfig).

Reply:
Sounds good, just wanted to make sure it wasn't an oops.